### PR TITLE
Add localhost:* and 127.0.0.1:* to cors

### PR DESCRIPTION
**Pull request by maxmwang**

### CODE REVIEW

**Review by mathhulk — APPROVED** [review]
should work pls double check

**Comment by kevinzwang** [comment]
Resolves #593

File: frontend/src/redux/actions.js [hunk 1]
```diff
@@ -16,6 +16,8 @@ import {
 	UPDATE_ENROLL_SELECTED
 } from './actionTypes';
 
+axios.defaults.baseURL = 'https://cors.berkeleytime.com';
```
Comment (kevinzwang) [inline frontend/src/redux/actions.js:19]: what is this for?

Comment (maxmwang) [reply frontend/src/redux/actions.js:19]: Oops that should not be there

**Review by kevinzwang — APPROVED** [review]
LGTM

**Pull request merged**